manifests/s/Synology/SurveillanceStationClient/2.2.1-2564/Synology.SurveillanceStationClient.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Synology.SurveillanceStationClient
+PackageVersion: 2.2.1-2564

```

Comment:
```suggestion
PackageVersion: 2.2.1.2564
```

Reply:
Thanks for reviewing the PR, this package version reflects the same naming scheme as the other synology packages already in the repo, such as: 
https://github.com/microsoft/winget-pkgs/blob/master/manifests/s/Synology/NoteStationClient/2.2.4-703/Synology.NoteStationClient.locale.en-US.yaml
I had figured keeping the versioning consistent might be better, however I'm not particularly worried either way. Please let me know if you'd still like to keep your suggested change.

Reply:
WinGet works by matching the metadata in the manifest to the actual values the installer writes to Registry (which are visible in Control Panel / Apps&Features). If the PackageVersion isn't matching the value written to Registry, WinGet will have trouble properly matching the package for list/upgrade scenarios. 

> I had figured keeping the versioning consistent might be better

It's more so related to what the Installer actually reports to Registry. In case of some Synology packages (example of Synology.DriveClient attached below), you can see the version is in form of `x.x.x.x` instead of `x.x.x-x`. In others it may be different. We'll have to use the registry value in the manifest


![image](https://github.com/user-attachments/assets/927bc1c8-a199-4f44-a872-08d42d83eb17)

![image](https://github.com/user-attachments/assets/c781c840-3956-4348-b652-cd026c197edb)

Reply:
Ok sweet, that's good to know for the future. Much appreciated.